MC/run/ANCHOR/2021/OCT/pass4/anchorMC.sh

```diff
@@ -104,8 +104,8 @@ baseargs="-tf ${NTIMEFRAMES} --split-id ${ALIEN_JDL_SPLITID:-1} --prod-split ${A
 # THIS NEEDS TO COME FROM OUTSIDE
 remainingargs="-gen pythia8 -proc cdiff -ns ${NSIGEVENTS}                                                                                                                 \
                -interactionRate ${INTERACTIONRATE}                                                                                                                        \
-               -confKey \"Diamond.width[2]=6.0;Diamond.width[0]=0.01;Diamond.width[1]=0.01;Diamond.position[0]=0.0;Diamond.position[1]=-0.035;Diamond.position[2]=0.41\"  \
-              --include-local-qc --include-analysis --mft-reco-full"
+               -confKey \"Diamond.width[2]=6.0;Diamond.width[0]=0.01;Diamond.width[1]=0.01;Diamond.position[0]=0.0;Diamond.position[1]=-0.035;Diamond.position[2]=0.41;MFTBase.buildAlignment=true\"  \
```

Comment:
I am not sure this is the best place. It might make more sense to switch this option to "on" directly in the O2 default code. (In any case one could always fetch ideal alignment CCDB ... whenever we don't want to have alignment). Otherwise, the better place is in `o2dpg_sim_workflow.py`